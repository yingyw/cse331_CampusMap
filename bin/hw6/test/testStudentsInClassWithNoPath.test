#Find a path from a student to another student using the full studentsInClass dataset.

LoadGraph studentsInClass studentsInClass.tsv

FindPath studentsInClass JERRY WISH