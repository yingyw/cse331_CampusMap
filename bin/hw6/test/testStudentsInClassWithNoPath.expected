#Find a path from a student to another student using the full studentsInClass dataset.

loaded graph studentsInClass

path from JERRY to WISH:
no path found